app/src/main/java/org/fossasia/susi/ai/rest/responses/susi/FiveStarSkillRatingResponse.kt

```diff
@@ -1,12 +1,26 @@
 package org.fossasia.susi.ai.rest.responses.susi
 
+import com.google.gson.annotations.Expose
+import com.google.gson.annotations.SerializedName
+
 /**
  *
  * Created by arundhati24 on 6/6/2018
  */
 class FiveStarSkillRatingResponse {
-    val ratings: Int = 0
+    @SerializedName("ratings")
+    @Expose
+    val ratings: Stars? = null
+
+    @SerializedName("session")
+    @Expose
     val session: Session? = null
+
+    @SerializedName("accepted")
+    @Expose
     val accepted: Boolean = false
+
+    @SerializedName("message")
+    @Expose
```

Comment:
I don't think you need SerializedName and Expose if property names are name, please confirm

Reply:
@iamareebjamal That's right. Thanks. Actually these annotations were not there initially in this file. Now I understand why these were not there. I will remove these as all the variable names used here are same as the keys.